assets/charts/control-plane/calico/templates/felixconfig.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  {{- if .Values.calico.failsafeInboundHostPorts }}
```

Comment:
```suggestion
  {{- if hasKey .Values.calico "failsafeInboundHostPorts" }}
```

Reply:
Thanks for the suggestion @invidian, but with this change, `[]` for packet it's not getting applied

Reply:
I am confused because `helm template` is getting rendered correctly also, 

```bash
➜ helm get values calico -n kube-system                                                                  
USER-SUPPLIED VALUES:
calico:
  blockedMetadataCIDRs:
  - 192.80.8.124/32
  cniImage: calico/cni:v3.15.2
  controllersImage: calico/kube-controllers:v3.15.2
  enableReporting: false
  encryptPodsTraffic: true
  failsafe_inbound_host_ports: []
  flexvolDriverImage: calico/pod2daemon-flexvol:v3.15.2
  image: calico/node:v3.15.2
  ipipEnabled: true
  networkEncapsulation: 'ipipMode: Always'
  networkIpAutodetectionMethod: can-reach=10.0.0.1
  networkMTU: 1480
  podCIDR: 10.2.0.0/16
  vxlanEnabled: false
```

it shows that `failsafe_inbound_host_ports` value is correctly getting rendered